assembly/assembly-ide-war/pom.xml

```diff
@@ -56,6 +56,16 @@
             <groupId>org.eclipse.che.core</groupId>
             <artifactId>che-core-commons-j2ee</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.che.core</groupId>
+            <artifactId>che-core-commons-logback</artifactId>
+            <exclusions>
+                <exclusion>
+                    <artifactId>che-core-api-core</artifactId>
```

Comment:
Why is it excluded?

Reply:
to make sure ide war doesn't contain che-core-api-core and all it unnecessary dependencies.

Reply:
Looks like a bit strange. It means that you're using classes from this dependency which don't use `api-core` classes.
But somebody can easily use it, and then Runtime exception will occur.
It's just thought. I'm OK to leave it as it is.

Reply:
Why we need logback in IDE war at all?

Reply:
@sleshchenko about exclusion.
We already have che-core-commons-logback module. To avoid exclusion we need another module with only one single class. When I selecting this approach I thought it would be less confusing then to have two modules.

Reply:
>>Why we need logback in IDE war at all?

Some filters are using slf4j-> logback logging.

Reply:
Well, it is more convenient right now, but we will have to remember that this exclusion is needed. And other projects that would extend Che also would need to know about this exclusion.

Reply:
That would not be a problem soon. Since that is not a case for Theia.
Do you propose to change this approach? Or it's a good enough? If change - then how?

Reply:
I'm ok with this approach, just thinking out loud